content/applications/hr/employees/new_employee.rst

```diff
@@ -7,13 +7,19 @@ When a new employee is hired, the first step is to create a new employee form. S
 employee form. Fill out the required information (underlined in bold) and any additional details,
 then click :guilabel:`Save`.
 
+When a new employee is hired, the first step is to create a new employee form. Starting in the
```

Comment:
I think it would be better to write this like this:
```suggestion
Aenean aliquet, magna sit amet vulputate vestibulum, magna tortor pharetra velit, n
```